aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -710,6 +719,25 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+
+						"kms_key_type": {
```

Comment:
This new attribute should be documented in `website/docs/r/kinesis_firehose_delivery_stream.html.markdown` 👍